### PR TITLE
fix(project.instances): disable deleting instance

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-18336
| License          | BSD 3-Clause

## Description

Disable link to instance details and delete instance when an instance is in deleting state

### CODE REVIEW

**Review by frenautvh — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instances.html [hunk 1]
```diff
@@ -48,9 +48,14 @@
             data-sortable="asc"
         >
             <a
+                data-ng-if="$row.status !== 'DELETING'"
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/instances/instances.html:51]: We could use a `isDeleting()` in the instance model if there is one

**Comment by ovh-cds** [comment]
CDS Report test#16078.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report test#16078.1 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by antleblanc** [comment]
@ganeshkumar1989  I'm rebasing this branch on `develop` in order to fix CI/CD errors reported.

**Comment by antleblanc** [comment]
Rebased successfully (waiting CI/CD reports).

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instances.html (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/pci/src/projects/project/instances/instances.html:51]: @marie-j This is what you had in mind, right?

See: #3850

File: packages/manager/modules/pci/src/projects/project/instances/instances.html [hunk 2]
```diff
@@ -137,6 +139,7 @@
             data-oui-popover-template="pci/projects/project/instances/help/backup.html"
             data-oui-popover-scope="$ctrl"
             data-placement="left-start"
+            data-ng-if="$row.status !== 'DELETING'"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/instances.html:142]:
```suggestion
            data-ng-if="!$row.isDeleting()'"
```

**Review by antleblanc — APPROVED** [review]

**Pull request merged**